blueocean-core-js/src/js/NotFound.jsx

```diff
@@ -11,11 +11,21 @@ const translate = i18nTranslator('blueocean-web');
 export default () => (
     <Fullscreen className="errorscreen not-found">
         <div className="message-box">
-            <h3>{translate('Not.found.heading', {
-                defaultValue: 'Page not found (404)',
-            })}</h3>
-            <div className="message">{translate('Not.found.message', { defaultValue: 'Jenkins could not find the page you were looking for. Check the URL for errors or press the back button.' })}</div>
-            <div className="actions"><Link to="/" className="btn">{translate('Open.dashboard', { defaultValue: 'Open Dashboard' })}</Link></div>
+            <h3>
+                {translate('Not.found.heading', {
```

Comment:
adding space like this _occasionally_ can be a problem

Reply:
React should just add a `<!-- some react magic cookie -->` But if it is a problem we can address it.